### PR TITLE
PR HAPI-101 building hdx-hapi image

### PR DESCRIPTION
**Pull request by alexandru-m-g**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM public.ecr.aws/unocha/python:3
+
+WORKDIR /srv/hapi
+
+COPY . .
+
+RUN apk add unit unit-python3 && \
+    apk --virtual .build-deps add \
+        git \
```
Comment (teodorescuserban) [inline Dockerfile:9]: duplicate.
spread them all out one per line, easier to read, although its build deps and get removed afterwards.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        build-base git python3-dev postgresql-dev && \
+    mkdir -p \
+        /etc/services.d/hapi \
+        /srv/cache \
```
Comment (teodorescuserban) [inline Dockerfile:13]: do you actually use these folders? if you do, let them in. if not, remove them
it looks like hxl-proxy to me.

File: logging.conf [hunk 3]
```diff
@@ -20,20 +20,20 @@ args=(sys.stdout,)
...
 
 [formatter_simpleFormatter]
-format=%(asctime)s %(levelname)-5.5s [%(name)s:%(lineno)d] %(message)s
+format=[%(process)d - %(thread)d] %(asctime)s %(levelname)-5.5s [%(name)s:%(lineno)d] %(message)s
 datefmt=
 
 [formatter_jsonFormatter]
-format = %(asctime)s %(levelname) %(threadName)s %(name)s %(lineno)d %(message)s %(funcName)s
+format = %(process)d %(thread)d %(asctime)s %(levelname) %(threadName)s %(name)s %(lineno)d %(message)s %(funcName)s
 class = pythonjsonlogger.jsonlogger.JsonFormatter
```
Comment (teodorescuserban) [inline logging.conf:39]:
```suggestion
class = pythonjsonlogger.jsonlogger.JsonFormatter

```

**Review by teodorescuserban — APPROVED** [review]
ok with the suggested changes.

File: Dockerfile (hunk 1 above)
Comment (alexandru-m-g) [reply Dockerfile:11]: Added the changes you suggested

**Pull request merged**